src/components/RegisterForm/StepOne.jsx

```diff
@@ -0,0 +1,66 @@
+import Button from '@mui/material/Button';
+import { Field, Form, Formik } from 'formik';
+import { TextField } from 'formik-material-ui';
+import * as Yup from 'yup';
+
+function StepOne(props) {
+  const handleSubmit = (values) => {
+    props.next(values);
+  };
+  const validate = Yup.object({
+    companyName: Yup.string().required('Required'),
+    companyLogin: Yup.string().min(5, 'Must be at least 5 characters').required('Required'),
+    companyPassword: Yup.string().min(7, 'Must be at least 7 characters').required('Required'),
+    companyPassword2: Yup.string()
+      .oneOf([Yup.ref('companyPassword'), null], 'Password must match')
+      .required('Required'),
+  });
+
+  return (
+    <Formik initialValues={props.data} onSubmit={handleSubmit} validationSchema={validate}>
+      {() => (
+        <Form>
+          <Field
+            fullWidth
+            sx={{ margin: '12px 0px 12px 0px' }}
+            name="companyName"
+            component={TextField}
+            label="Company Name"
+          />
+
+          <Field
+            fullWidth
+            sx={{ marginBottom: '12px' }}
+            name="companyLogin"
+            component={TextField}
+            label="Company Login"
+          />
+
+          <Field
+            fullWidth
+            sx={{ marginBottom: '12px' }}
+            name="companyPassword"
+            component={TextField}
+            type="password"
+            label="Company Password"
+          />
+
+          <Field
+            fullWidth
+            sx={{ marginBottom: '12px' }}
+            name="companyPassword2"
```

Comment:
Does it have for `backend` any differences if it is called `companyPassword` or `password`? If not, then still `companyPasswordRepeat` or `repeatCompanyPassword` would be more verbose